executor/insert_test.go

```diff
@@ -595,6 +595,27 @@ func TestInsertWithAutoidSchema(t *testing.T) {
 	// test for inserting multiple values
 	tk.MustExec(`create table t8(id int primary key auto_increment, n int);`)
 
+	testInsertWithAutoidSchema(t, tk)
+}
+
+func TestInsertWithAutoidSchemaCache(t *testing.T) {
```

Comment:
Can we create a ```executor/auto_id_cache``` folder to save the test? if the same folder saves too much test to slow the speed of the test when using the bazel.

Reply:
Done